The patch removes the test that relies on bpf and seccomp filters.
The filters are available in Linux kernel 3.5 or newer.
Moreover, the test was only tested on i386 and x86_64 platforms,
and as a result it simply fails on sparc machines.

Initial commit that introduced this test:
https://github.com/memcached/memcached/commit/78c260a2ea8a3662720562ef2c0364eac36dfa4a

The patch is not suitable for upstream.

--- memcached-1.5.14/t/misbehave.t	2019-04-27 09:52:36.000000000 +0000
+++ /dev/null	2019-05-13 15:41:05.000000000 +0000
@@ -1,28 +0,0 @@
-#!/usr/bin/perl
-
-use strict;
-use Test::More;
-use FindBin qw($Bin);
-use Socket qw(MSG_PEEK MSG_DONTWAIT);
-use lib "$Bin/lib";
-use MemcachedTest;
-
-if (!enabled_tls_testing() && supports_drop_priv()) {
-    plan tests => 1;
-} else {
-    plan skip_all => 'Privilege drop not supported';
-    exit 0;
-}
-
-my $server = new_memcached('-o drop_privileges');
-my $sock = $server->sock;
-
-print $sock "misbehave\r\n";
-sleep(1);
-
-# check if the socket is dead now
-my $buff;
-my $ret = recv($sock, $buff, 1, MSG_PEEK | MSG_DONTWAIT);
-is($ret, undef, "did not allow misbehaving");
-
-$server->DESTROY();
